back/app/models/user.rb

```diff
@@ -216,16 +211,34 @@ def full_name
     [first_name, last_name].compact.join(' ')
   end
 
-  def admin?
-    roles.any? { |r| r['type'] == 'admin' }
+  def highest_role
```

Comment:
Centralized definition in the core for now. Before it was overridden by the project-folders engine.